### PR TITLE
Update MainCLI.java

### PR DESCRIPTION
**Pull request by vlOd2**
Fix https://github.com/MCPHackers/RetroMCP-Java/issues/24

### CODE REVIEW

File: src/main/java/org/mcphackers/mcp/main/MainCLI.java [hunk 1]
```diff
@@ -48,12 +47,28 @@ public class MainCLI extends MCP {
...
+		    	String jarPath = MainCLI.class.getProtectionDomain().getCodeSource().getLocation().toURI().getPath();
+		    	String homeDir = System.getProperty("user.home");
+		    	String currentPath = Paths.get("").toAbsolutePath().toString();
+		    	String correctPath = Paths.get(jarPath).getParent().toString();
+
+		    	// HACK: This is dirty hack to detect if the path is broken
+		    	if (currentPath.equals(homeDir)) {
+				String currOs = System.getProperty("os.name");
+				if (!currOs.contains("Windows")) {
+			    		System.loadLibrary("c");
```
Comment (ThexXTURBOXx) [inline src/main/java/org/mcphackers/mcp/main/MainCLI.java:67]: Why is this needed in the first place?

File: src/main/java/org/mcphackers/mcp/main/MainCLI.java [hunk 2]
```diff
@@ -48,12 +47,28 @@ public class MainCLI extends MCP {
...
 		else {
+		    	String jarPath = MainCLI.class.getProtectionDomain().getCodeSource().getLocation().toURI().getPath();
+		    	String homeDir = System.getProperty("user.home");
+		    	String currentPath = Paths.get("").toAbsolutePath().toString();
+		    	String correctPath = Paths.get(jarPath).getParent().toString();
+
+		    	// HACK: This is dirty hack to detect if the path is broken
+		    	if (currentPath.equals(homeDir)) {
+				String currOs = System.getProperty("os.name");
+				if (!currOs.contains("Windows")) {
```
Comment (ThexXTURBOXx) [inline src/main/java/org/mcphackers/mcp/main/MainCLI.java:66]: Wrong indentation

File: src/main/java/org/mcphackers/mcp/main/MainCLI.java [hunk 3]
```diff
@@ -48,12 +47,28 @@ public class MainCLI extends MCP {
 	private String[] progressStrings;
 	private String[] progressBarNames;
 
+    	public static native int chdir(String path);
+
 	public static void main(String[] args) throws Exception {
-		if(System.console() != null || FORCE_CONSOLE) {
+        	if (System.console() != null && System.getProperty("doNotUseConsole") == null) {
```
Comment (ThexXTURBOXx) [inline src/main/java/org/mcphackers/mcp/main/MainCLI.java:53]: Wrong indentation

**Review by ThexXTURBOXx — CHANGES_REQUESTED** [review]

**Comment by ThexXTURBOXx** [comment]
Sorry, just noticed that this is marked as will-not-fix

**Comment by Lassebq** [comment]
Yeah, I forgot to close pull request to that issue, doing that now

**Pull request closed without merging**